grammar.pegjs

```diff
@@ -33,13 +33,27 @@ selectors = s:selector ss:(_ "," _ selector)* {
   return [s].concat(ss.map(function (s) { return s[3]; }));
```

Comment:
If we get rid of the old syntax entirely and break the selector API, can some of the grammar be removed?